### ISSUE TITLE
Conditions not working

### ISSUE THREAD

**Issue by mark-chief** [verbb/formie#728]
**Description**
Trying to use the "is greater than" or "is less than", "is not" with a radio button, where the values are numbers 1-10.

<img width="903" alt="Screenshot 2022-01-24 at 17 21 17" src="https://user-images.githubusercontent.com/18699249/150832591-1f746901-ff11-4129-b900-7770edb68641.png">

<img width="904" alt="Screenshot 2022-01-24 at 17 22 53" src="https://user-images.githubusercontent.com/18699249/150832844-dcee071d-c5ee-4f30-96cd-ebfad402b3c8.png">

**Steps to reproduce**
1. Create text box field and create conditions based on values in radio button
2.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.7
- Craft version: 3.7.24
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mark-chief** [comment]
Thanks, doing the above, and then composer update gives me huge error that starts with
```
[UnexpectedValueException]                                                   
  Unknown package has no name defined 
```

**Comment by engram-design** [comment]
That's strange! Can you shoot me your `composer.json` file?

**Comment by mark-chief** [comment]
```
{
  "require": {
    "craftcms/cms": "3.7.27.2",
    "craftcms/redactor": "2.8.8",
    "doublesecretagency/craft-cpcss": "2.4.0",
    "kisonay/craft-twig-imagebase64": "1.0.4",
    "nystudio107/craft-cookies": "1.1.15",
    "putyourlightson/craft-sprig": "1.10.4",
    "solspace/craft-freeform": "3.12.11",
    "verbb/cp-nav": "3.0.0",
    "verbb/formie": "dev-craft-3 as 1.5.7",
    "vlucas/phpdotenv": "^3.4.0"
  },
  "autoload": {
    "psr-4": {
      "modules\\": "modules/"
    }
  },
  "config": {
    "sort-packages": true,
    "optimize-autoloader": true,
    "platform": {
      "php": "7.4"
    }
  },
  "scripts": {
    "post-root-package-install": [
      "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
    ]
  }
}

```
probably something I am doing wrong!

**Comment by engram-design** [comment]
Seems to be working for me!

```
$ composer update
Loading composer repositories with package information
Updating dependencies
Lock file operations: 130 installs, 0 updates, 0 removals
  - Locking cebe/markdown (1.2.1)
  - Locking commerceguys/addressing (v1.2.2)
  - Locking composer/ca-bundle (1.3.1)
  - Locking composer/composer (2.1.9)
  - Locking composer/metadata-minifier (1.0.0)
  - Locking composer/pcre (1.0.1)
  - Locking composer/semver (3.2.7)
  - Locking composer/spdx-licenses (1.5.6)
  - Locking composer/xdebug-handler (2.0.4)
  - Locking craftcms/cms (3.7.27.2)
  - Locking craftcms/oauth2-craftid (1.0.0.1)
  - Locking craftcms/plugin-installer (1.5.7)
  - Locking craftcms/redactor (2.8.8)
  - Locking craftcms/server-check (1.2.3)
  - Locking creocoder/yii2-nested-sets (0.9.0)
  - Locking defuse/php-encryption (v2.3.1)
  - Locking doctrine/collections (1.6.8)
  - Locking doctrine/lexer (1.2.2)
  - Locking dompdf/dompdf (v1.1.1)
  - Locking doublesecretagency/craft-cpcss (2.4.0)
  - Locking egulias/email-validator (3.1.2)
  - Locking elvanto/litemoji (3.0.1)
  - Locking enshrined/svg-sanitize (0.14.1)
  - Locking ezyang/htmlpurifier (v4.14.0)
  - Locking fakerphp/faker (v1.18.0)
  - Locking firebase/php-jwt (v5.5.1)
  - Locking giggsey/libphonenumber-for-php (8.12.41)
  - Locking giggsey/locale (2.1)
  - Locking guzzlehttp/guzzle (7.4.1)
  - Locking guzzlehttp/oauth-subscriber (0.6.0)
  - Locking guzzlehttp/promises (1.5.1)
  - Locking guzzlehttp/psr7 (2.1.0)
  - Locking hashids/hashids (4.1.0)
  - Locking html2text/html2text (4.3.1)
  - Locking justinrainbow/json-schema (5.2.11)
  - Locking kisonay/craft-twig-imagebase64 (1.0.4)
  - Locking laminas/laminas-escaper (2.9.0)
  - Locking laminas/laminas-feed (2.16.0)
  - Locking laminas/laminas-stdlib (3.7.1)
  - Locking league/flysystem (1.1.9)
  - Locking league/html-to-markdown (4.10.0)
  - Locking league/mime-type-detection (1.9.0)
  - Locking league/oauth1-client (v1.10.0)
  - Locking league/oauth2-client (2.6.1)
  - Locking league/oauth2-google (3.0.4)
  - Locking maennchen/zipstream-php (2.1.0)
  - Locking markbaker/complex (3.0.1)
  - Locking markbaker/matrix (3.0.0)
  - Locking mikehaertl/php-shellcommand (1.6.4)
  - Locking monolog/monolog (2.3.5)
  - Locking myclabs/php-enum (1.8.3)
  - Locking nesbot/carbon (2.56.0)
  - Locking nystudio107/craft-cookies (1.1.15)
  - Locking opis/closure (3.6.2)
  - Locking paragonie/random_compat (v9.99.100)
  - Locking phenx/php-font-lib (0.5.4)
  - Locking phenx/php-svg-lib (0.3.4)
  - Locking phpdocumentor/reflection-common (2.2.0)
  - Locking phpdocumentor/reflection-docblock (5.3.0)
  - Locking phpdocumentor/type-resolver (1.6.0)
  - Locking phpoffice/phpspreadsheet (1.21.0)
  - Locking phpoption/phpoption (1.8.1)
  - Locking pixelandtonic/imagine (1.2.4.2)
  - Locking psr/cache (1.0.1)
  - Locking psr/container (2.0.2)
  - Locking psr/http-client (1.0.1)
  - Locking psr/http-factory (1.0.1)
  - Locking psr/http-message (1.0.1)
  - Locking psr/log (1.1.4)
  - Locking psr/simple-cache (1.0.1)
  - Locking putyourlightson/craft-sprig (1.10.4)
  - Locking putyourlightson/craft-sprig-core (1.1.6)
  - Locking ralouphie/getallheaders (3.0.3)
  - Locking react/promise (v2.8.0)
  - Locking sabberworm/php-css-parser (8.4.0)
  - Locking seld/cli-prompt (1.0.4)
  - Locking seld/jsonlint (1.8.3)
  - Locking seld/phar-utils (1.2.0)
  - Locking solspace/craft-freeform (3.12.11)
  - Locking solspace/craft3-commons (1.0.25)
  - Locking stripe/stripe-php (v7.111.0)
  - Locking swiftmailer/swiftmailer (v6.3.0)
  - Locking symfony/cache (v5.4.2)
  - Locking symfony/cache-contracts (v2.5.0)
  - Locking symfony/console (v5.4.2)
  - Locking symfony/deprecation-contracts (v2.5.0)
  - Locking symfony/expression-language (v5.4.0)
  - Locking symfony/filesystem (v5.4.0)
  - Locking symfony/finder (v5.4.2)
  - Locking symfony/polyfill-ctype (v1.24.0)
  - Locking symfony/polyfill-iconv (v1.24.0)
  - Locking symfony/polyfill-intl-grapheme (v1.24.0)
  - Locking symfony/polyfill-intl-idn (v1.24.0)
  - Locking symfony/polyfill-intl-normalizer (v1.24.0)
  - Locking symfony/polyfill-mbstring (v1.24.0)
  - Locking symfony/polyfill-php72 (v1.24.0)
  - Locking symfony/polyfill-php73 (v1.24.0)
  - Locking symfony/polyfill-php80 (v1.24.0)
  - Locking symfony/process (v5.4.2)
  - Locking symfony/property-access (v5.4.2)
  - Locking symfony/property-info (v5.4.2)
  - Locking symfony/service-contracts (v1.1.2)
  - Locking symfony/string (v5.4.2)
  - Locking symfony/translation (v5.4.2)
  - Locking symfony/translation-contracts (v2.5.0)
  - Locking symfony/var-exporter (v5.4.2)
  - Locking symfony/yaml (v5.4.2)
  - Locking thenetworg/oauth2-azure (v2.0.1)
  - Locking true/punycode (v2.1.1)
  - Locking twig/twig (v2.14.10)
  - Locking verbb/base (1.0.4)
  - Locking verbb/cp-nav (3.0.0)
  - Locking verbb/formie (dev-craft-3 f239700)
  - Locking vlucas/phpdotenv (v3.6.10)
  - Locking voku/anti-xss (4.1.35)
  - Locking voku/arrayy (7.8.14)
  - Locking voku/email-check (3.1.0)
  - Locking voku/portable-ascii (1.6.1)
  - Locking voku/portable-utf8 (6.0.1)
  - Locking voku/stop-words (2.0.1)
  - Locking voku/stringy (6.5.0)
  - Locking voku/urlify (5.0.6)
  - Locking webmozart/assert (1.10.0)
  - Locking webonyx/graphql-php (v14.4.1)
  - Locking yii2tech/ar-softdelete (1.0.4)
  - Locking yiisoft/yii2 (2.0.44)
  - Locking yiisoft/yii2-composer (2.0.10)
  - Locking yiisoft/yii2-debug (2.1.18)
  - Locking yiisoft/yii2-queue (2.3.3)
  - Locking yiisoft/yii2-swiftmailer (2.1.3)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 130 installs, 0 updates, 0 removals
  - Installing craftcms/plugin-installer (1.5.7): Extracting archive
  - Installing yiisoft/yii2-composer (2.0.10): Extracting archive
  - Installing cebe/markdown (1.2.1): Extracting archive
  - Installing composer/ca-bundle (1.3.1): Extracting archive
  - Installing composer/metadata-minifier (1.0.0): Extracting archive
  - Installing composer/semver (3.2.7): Extracting archive
  - Installing composer/spdx-licenses (1.5.6): Extracting archive
  - Installing psr/log (1.1.4): Extracting archive
  - Installing composer/pcre (1.0.1): Extracting archive
  - Installing composer/xdebug-handler (2.0.4): Extracting archive
  - Installing paragonie/random_compat (v9.99.100): Extracting archive
  - Installing symfony/deprecation-contracts (v2.5.0): Extracting archive
  - Installing psr/http-message (1.0.1): Extracting archive
  - Installing psr/http-client (1.0.1): Extracting archive
  - Installing ralouphie/getallheaders (3.0.3): Extracting archive
  - Installing psr/http-factory (1.0.1): Extracting archive
  - Installing guzzlehttp/psr7 (2.1.0): Extracting archive
  - Installing guzzlehttp/promises (1.5.1): Extracting archive
  - Installing guzzlehttp/guzzle (7.4.1): Extracting archive
  - Installing league/oauth2-client (2.6.1): Extracting archive
  - Installing craftcms/oauth2-craftid (1.0.0.1): Extracting archive
  - Installing ezyang/htmlpurifier (v4.14.0): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.24.0): Extracting archive
  - Installing symfony/polyfill-ctype (v1.24.0): Extracting archive
  - Installing craftcms/cms (3.7.27.2): Extracting archive
  - Installing yiisoft/yii2 (2.0.44): Extracting archive
  - Installing symfony/polyfill-php72 (v1.24.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.24.0): Extracting archive
  - Installing symfony/polyfill-intl-idn (v1.24.0): Extracting archive
  - Installing symfony/polyfill-iconv (v1.24.0): Extracting archive
  - Installing doctrine/lexer (1.2.2): Extracting archive
  - Installing egulias/email-validator (3.1.2): Extracting archive
  - Installing swiftmailer/swiftmailer (v6.3.0): Extracting archive
  - Installing yiisoft/yii2-swiftmailer (2.1.3): Extracting archive
  - Installing symfony/polyfill-php80 (v1.24.0): Extracting archive
  - Installing symfony/process (v5.4.2): Extracting archive
  - Installing yiisoft/yii2-queue (2.3.3): Extracting archive
  - Installing opis/closure (3.6.2): Extracting archive
  - Installing yiisoft/yii2-debug (2.1.18): Extracting archive
  - Installing yii2tech/ar-softdelete (1.0.4): Extracting archive
  - Installing webonyx/graphql-php (v14.4.1): Extracting archive
  - Installing voku/stop-words (2.0.1): Extracting archive
  - Installing voku/portable-ascii (1.6.1): Extracting archive
  - Installing symfony/polyfill-intl-grapheme (v1.24.0): Extracting archive
  - Installing voku/portable-utf8 (6.0.1): Extracting archive
  - Installing voku/urlify (5.0.6): Extracting archive
  - Installing voku/email-check (3.1.0): Extracting archive
  - Installing webmozart/assert (1.10.0): Extracting archive
  - Installing phpdocumentor/reflection-common (2.2.0): Extracting archive
  - Installing phpdocumentor/type-resolver (1.6.0): Extracting archive
  - Installing phpdocumentor/reflection-docblock (5.3.0): Extracting archive
  - Installing voku/arrayy (7.8.14): Extracting archive
  - Installing voku/anti-xss (4.1.35): Extracting archive
  - Installing defuse/php-encryption (v2.3.1): Extracting archive
  - Installing voku/stringy (6.5.0): Extracting archive
  - Installing twig/twig (v2.14.10): Extracting archive
  - Installing true/punycode (v2.1.1): Extracting archive
  - Installing symfony/yaml (v5.4.2): Extracting archive
  - Installing seld/cli-prompt (1.0.4): Extracting archive
  - Installing pixelandtonic/imagine (1.2.4.2): Extracting archive
  - Installing mikehaertl/php-shellcommand (1.6.4): Extracting archive
  - Installing league/mime-type-detection (1.9.0): Extracting archive
  - Installing league/flysystem (1.1.9): Extracting archive
  - Installing laminas/laminas-stdlib (3.7.1): Extracting archive
  - Installing laminas/laminas-escaper (2.9.0): Extracting archive
  - Installing laminas/laminas-feed (2.16.0): Extracting archive
  - Installing enshrined/svg-sanitize (0.14.1): Extracting archive
  - Installing elvanto/litemoji (3.0.1): Extracting archive
  - Installing creocoder/yii2-nested-sets (0.9.0): Extracting archive
  - Installing craftcms/server-check (1.2.3): Extracting archive
  - Installing symfony/finder (v5.4.2): Extracting archive
  - Installing symfony/filesystem (v5.4.0): Extracting archive
  - Installing symfony/string (v5.4.2): Extracting archive
  - Installing symfony/service-contracts (v1.1.2): Extracting archive
  - Installing symfony/polyfill-php73 (v1.24.0): Extracting archive
  - Installing symfony/console (v5.4.2): Extracting archive
  - Installing seld/phar-utils (1.2.0): Extracting archive
  - Installing seld/jsonlint (1.8.3): Extracting archive
  - Installing react/promise (v2.8.0): Extracting archive
  - Installing justinrainbow/json-schema (5.2.11): Extracting archive
  - Installing composer/composer (2.1.9): Extracting archive
  - Installing craftcms/redactor (2.8.8): Extracting archive
  - Installing doctrine/collections (1.6.8): Extracting archive
  - Installing doublesecretagency/craft-cpcss (2.4.0): Extracting archive
  - Installing firebase/php-jwt (v5.5.1): Extracting archive
  - Installing giggsey/locale (2.1): Extracting archive
  - Installing kisonay/craft-twig-imagebase64 (1.0.4): Extracting archive
  - Installing myclabs/php-enum (1.8.3): Extracting archive
  - Installing maennchen/zipstream-php (2.1.0): Extracting archive
  - Installing markbaker/complex (3.0.1): Extracting archive
  - Installing markbaker/matrix (3.0.0): Extracting archive
  - Installing nystudio107/craft-cookies (1.1.15): Extracting archive
  - Installing phenx/php-font-lib (0.5.4): Extracting archive
  - Installing sabberworm/php-css-parser (8.4.0): Extracting archive
  - Installing phenx/php-svg-lib (0.3.4): Extracting archive
  - Installing psr/container (2.0.2): Extracting archive
  - Installing psr/simple-cache (1.0.1): Extracting archive
  - Installing putyourlightson/craft-sprig-core (1.1.6): Extracting archive
  - Installing putyourlightson/craft-sprig (1.10.4): Extracting archive
  - Installing symfony/property-info (v5.4.2): Extracting archive
  - Installing symfony/property-access (v5.4.2): Extracting archive
  - Installing stripe/stripe-php (v7.111.0): Extracting archive
  - Installing monolog/monolog (2.3.5): Extracting archive
  - Installing solspace/craft3-commons (1.0.25): Extracting archive
  - Installing phpoffice/phpspreadsheet (1.21.0): Extracting archive
  - Installing symfony/translation-contracts (v2.5.0): Extracting archive
  - Installing symfony/translation (v5.4.2): Extracting archive
  - Installing nesbot/carbon (2.56.0): Extracting archive
  - Installing hashids/hashids (4.1.0): Extracting archive
  - Installing solspace/craft-freeform (3.12.11): Extracting archive
  - Installing symfony/var-exporter (v5.4.2): Extracting archive
  - Installing psr/cache (1.0.1): Extracting archive
  - Installing symfony/cache-contracts (v2.5.0): Extracting archive
  - Installing symfony/cache (v5.4.2): Extracting archive
  - Installing verbb/base (1.0.4): Extracting archive
  - Installing verbb/cp-nav (3.0.0): Extracting archive
  - Installing thenetworg/oauth2-azure (v2.0.1): Extracting archive
  - Installing symfony/expression-language (v5.4.0): Extracting archive
  - Installing league/oauth2-google (3.0.4): Extracting archive
  - Installing league/oauth1-client (v1.10.0): Extracting archive
  - Installing league/html-to-markdown (4.10.0): Extracting archive
  - Installing html2text/html2text (4.3.1): Extracting archive
  - Installing guzzlehttp/oauth-subscriber (0.6.0): Extracting archive
  - Installing giggsey/libphonenumber-for-php (8.12.41): Extracting archive
  - Installing fakerphp/faker (v1.18.0): Extracting archive
  - Installing dompdf/dompdf (v1.1.1): Extracting archive
  - Installing commerceguys/addressing (v1.2.2): Extracting archive
  - Installing verbb/formie (dev-craft-3 f239700): Extracting archive
  - Installing phpoption/phpoption (1.8.1): Extracting archive
  - Installing vlucas/phpdotenv (v3.6.10): Extracting archive
50 package suggestions were added by new dependencies, use `composer suggest` to see details.
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.
Package yii2tech/ar-softdelete is abandoned, you should avoid using it. No replacement was suggested.
Generating optimized autoload files
66 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
```

Do you usually update your plugins via composer, or use the plugin store? *Maybe* worth trying it again - or no stress as I'll probably release a new version in the next day or so if you're up to waiting.

**Comment by engram-design** [comment]
Fixed in [1.5.8](https://github.com/verbb/formie/releases/tag/1.5.8)
